contribute/design/README.md

```diff
@@ -1,29 +1,58 @@
 # Overview
 
-OpenEBS is a open source storage platform delivering containerized block storage for containers. OpenEBS aims at making storage instrastructure programmable, easy-to-use, consumed by applications via self-service. OpenEBS takes storage software upgrade seriously.
+OpenEBS is a open source storage platform delivering containerized block storage for containers. 
 
-The storage controller functionality is delivered as containers - openebs/jiva. An OpenEBS Volume comprises of one or more containers working as a clustered microservice, providing block storage to other containers. OpenEBS Volumes are fully isolated user space storage engines that present the block storage via iSCSI and persist the data onto the storage attached to Nodes (aka Docker Hosts). The storage attached to the docker hosts can either be directly attached disks or cloud volumes depending on where the Container Cluster is deployed. OpenEBS uses *Maya*, a storage orchestration engine that helps in managing local and remote storage, integrate into the container orchestrators like Kubernetes, Docker Swarm, Nomad etc., manages QoS policies for OpenEBS Volumes.
+Yes! The storage controller functionality is itself, delivered as containers. An OpenEBS Volume comprises of one or more containers working as a clustered microservice, providing block storage to other containers. This micro-services based architecture for storage controller functionality - where each Volume data is being served by its own set of containers, as opposed to a single monolothic storage controller software serving multiples volumes is what differentiates OpenEBS from traditional storage appliances.
```

Comment:
where each volume data is served by its own set of containers as against a single monolithic storage controller serving multiple volumes. This is what differentiates OpenEBS from traditional storage appliances.

Reply:
done